modules/web/client/src/main/scala/observe/ui/components/sequence/SequenceTable.scala

```diff
@@ -269,15 +273,31 @@ private sealed trait SequenceTableBuilder[S: Eq, D <: DynamicConfig: Eq]
                 case _                                    =>
                   TagMod.empty
 
-        PrimeAutoHeightVirtualizedTable(
-          table,
-          estimateSize = _ => 25.toPx,
-          overscan = 8,
-          containerRef = resize.ref,
-          tableMod = TagMod(tableStyle),
-          rowMod = row => computeRowMods(row.original),
-          headerCellMod = computeHeaderCellMods,
-          cellMod = computeCellMods
+        React.Fragment(
+          // Render the expand all button in the ObsHeader
+          Option(dom.document.getElementById("sequence-table-expand-all")).map {
```

Comment:
Can we use a `ref` for the DOM Node? We would have to define it in a common ancestor.